chore: insert OwnerReferences param to pipelines (#452)
kubevirt/kubevirt-tekton-tasks: release/pipelines/windows-efi-installer/README.md

## Cancelling/Deleting PipelineRuns

When running the example Pipelines, they create temporary objects (DataVolumes, VirtualMachines, etc.). Each Pipeline has its own clean up system which should keep the cluster clean from leftovers. In case user hard deletes or cancels running PipelineRun, the PipelineRun will not clean temporary objects and objects will stay in the cluster. To prevent this behaviour, cancel the [PipelineRun gracefully](https://tekton.dev/docs/pipelines/pipelineruns/#gracefully-cancelling-a-pipelinerun). It triggers special Tasks, which remove temporary objects and keep only result DataSource/DataVolume/PVC.

> Each object created by the Pipeline has OwnerReference to the Pod which created them (result DataSource and DataVolume do not have it). If for some reasons the clean up steps are not triggered, delete the PipelineRun and all leftover objects created by the Pipeline will be deleted.

Reviewer: ```suggestion
Each object created by the Pipeline has OwnerReference to the Pod which created them (result DataSource and DataVolume do not have it). In the case that the clean up steps are not triggered, by deleting the PipelineRun all leftover objects created by the Pipeline will be deleted.
```

> windows-efi-installer Pipeline generates for each PipelineRun new source DataVolume which contains imported ISO file. This DataVolume has generated name and is deleted after Pipeline succeeds. However, the created PVC will stay in cluster, but it will have terminating state. It will wait, until pipelinRun is deleted. This behaviour is caused by a fact, that PVC is mounted into modify-windows-iso TaskRun pod and PVC can be deleted only when the pod does not exist.

Reviewer: ```suggestion
windows-efi-installer Pipeline generates for each PipelineRun a new source DataVolume which contains the imported ISO file. This DataVolume has a generated name, the DataVolume will be deleted if the PipelineRun succeeds. However, the created PVC will stay in cluster, but it will be in terminating state. It will wait until pipelinRun is deleted. This is due to the fact that the PVC is mounted into modify-windows-iso TaskRun pod and PVC can be deleted only when the pod does not exist.
```
